CHANGELOG.md

```diff
@@ -115,83 +137,86 @@ The following patterns are _not_ considered violations:
 .Foo {
   position: relative;
   display: block;
-  $foo: 'foo';
+  $foo: "foo";
```

Comment:
Aren't we using single quotes?

Reply:
Just noticed prettier isn't configured in this repo. My VSCode environment applied the default prettier settings to the onsave which lead to this. The right thing here is to revert my 'onsave' changes. Adding and configuring prettier in this repo should be in a separate pr.